.gitignore

```diff
@@ -0,0 +1,7272 @@
+
```

Comment:
Can you try just specifying `env/` in `.gitignore`? I know I've had issues with this before so it might not work, but my current repo's `.gitignore` is just:
```
*.ttl
dist/
env/
```

Reply:
Otherwise this PR all looks great! Thanks!